components/compliance-service/config/config_mgmt.go

```diff
@@ -4,27 +4,16 @@ import (
 	"io/ioutil"
 	"os"
 
+	project_update_lib "github.com/chef/automate/lib/authz"
```

Comment:
Updating to use the common config struct.

Reply:
nitpick: Any reason to not just leave this as `authz`?

Reply:
I only thought that the `project_update_lib` was a better description of the package.